src/alpine/3.20/helix/amd64/Dockerfile

```diff
@@ -0,0 +1,86 @@
+FROM amd64/alpine:3.20 as msquic
+
+# build MsQuic as we don't have packages
+RUN apk update && apk add --no-cache && \
+    apk add \
+        cmake \
+        g++ \
+        gcc \
+        git \
+        numactl-dev \
+        linux-headers \
+        lttng-ust-dev \
+        make \
+        musl-dev \
+        openssl-dev \
+        perl
+
+RUN git clone --depth 1 --single-branch --branch v2.3.5 --recursive https://github.com/microsoft/msquic /tmp/msquic
+
+WORKDIR /tmp/msquic
+
+RUN cmake -B build/linux/x64_Release_openssl3 \
+       -DQUIC_OUTPUT_DIR=/tmp/msquic/artifacts/bin/linux/x64_Release_openssl3 \
+       -DCMAKE_BUILD_TYPE=Release \
+       -DQUIC_TLS=openssl3 \
+       -DQUIC_ENABLE_LOGGING=true \
+       -DQUIC_USE_SYSTEM_LIBCRYPTO=true \
+       -DQUIC_BUILD_TOOLS=off \
+       -DQUIC_BUILD_TEST=off \
+       -DQUIC_BUILD_PERF=off && \
+    cmake --build build/linux/x64_Release_openssl3  --config Release && \
+    cmake --install build/linux/x64_Release_openssl3 --prefix /msquic
+
+FROM alpine:3.20
+# Install .NET and test dependencies
+RUN apk update && apk add --no-cache \
+    bash \
+    coreutils \
+    curl \
+    icu-data-full \
+    icu-libs \
+    iputils \
+    krb5-libs \
+    lldb \
+    llvm \
+    lttng-ust \
+    musl-locales \
+    numactl \
+    openssl \
+    python3 \
+    python3-dev \
+    py3-pip \
+    sudo \
+    tzdata
```

Comment:
These should be intended to illustrate they're continuations of the previous line's command. Please apply that formatting to other usages in this PR.